packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page.tsx

```diff
@@ -85,31 +87,39 @@ const Page: NextPageWithLayout = () => {
 
   const propertyTypes = useRemotePropertyTypes();
 
+  const getSchemaFromEditorForm = (
```

Comment:
This function can be at the top level – it doesn't use anything from the above scope

Reply:
Done in ecb53db93b148621b61f65bcbe764c2fd263e1ce